context/src/env/parser.rs

```diff
@@ -481,7 +502,12 @@ impl From<(&str, Option<usize>, Option<GitLabMergeRequestEventType>)> for Branch
             BranchClass::PullRequest
         } else if branch_name.starts_with("remotes/pull/") || branch_name.starts_with("pull/") {
             BranchClass::PullRequest
-        } else if matches!(branch_name, "master" | "main") {
+        } else if stable_branches
+            .filter(|branches| !branches.is_empty())
+            .unwrap_or(vec![String::from("main"), String::from("master")])
+            .iter()
+            .any(|branch| branch == branch_name)
```

Comment:
if stable branches is an empty vector or is none, we should default to main/master